src/domain/transaction/evm/status.rs

```diff
@@ -111,11 +111,35 @@ where
             )));
         }
 
+        let evm_data = tx.network_data.get_evm_transaction_data()?;
+
+        // Check if network lacks mempool (like Arbitrum) - skip resubmission for such networks
+        let network_model = self
+            .network_repository()
+            .get_by_chain_id(NetworkType::Evm, evm_data.chain_id)
+            .await?
+            .ok_or(TransactionError::UnexpectedError(format!(
+                "Network with chain id {} not found",
+                evm_data.chain_id
+            )))?;
+
+        let network = EvmNetwork::try_from(network_model).map_err(|e| {
+            TransactionError::UnexpectedError(format!(
+                "Error converting network model to EvmNetwork: {}",
+                e
+            ))
+        })?;
+
+        if network.lacks_mempool() {
```

Comment:
We should resubmit, we just resubmit without bumping gas prices but just using market prices.

This is used in `handle_submitted_state` fn which produces async job that us handled via `resubmit_transaction`